documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -14,6 +14,26 @@ You must specify a name for cluster-scoped CRs.
 You must specify both a name and a namespace for namespace-scoped CRs.
 ====
 
+As a Technology Preview, {project-short} supports migrations using OpenStack source providers.
+
+[IMPORTANT]
+====
+Migration using OpenStack source providers is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+// [NOTE]
+// ====
+// Migration using OpenStack source providers only supports VMs whose volume snapshots are supported by OpenStack Block Storage (Cinder). Migrations for VMs that use OpenStack Swift at all or use only OpenStack Glance for backend storage are not currently supported.
```

Comment:
I think this needs to be rephased, volumes are a Cinder component, so the VM is required to have only volumes as opposed to using a Glance image directly

Reply:
I suggest we use the note we used for the prerequisites throughout the guide:

[NOTE]
====
Migration using {osp} source providers only supports VMs that use only Cinder volumes.
====

Reply:
[NOTE]
====
Migration using {osp} source providers only supports VMs that use only Cinder volumes.
====